learning/tour-of-beam/learning-content/introduction/introduction-concepts/runner-concepts/description.md

```diff
@@ -95,7 +136,21 @@ java -jar target/beam-examples-bundled-1.0.0.jar \
   --region=<GCP_REGION> \
   --tempLocation=gs://<YOUR_GCS_BUCKET>/temp/
 ```
+{{end}}
+{{if (eq .Sdk "python")}}
+```
+# As part of the initial setup, install Google Cloud Platform specific extra components.
+pip install apache-beam[gcp]
+python -m apache_beam.examples.wordcount --input gs://dataflow-samples/shakespeare/kinglear.txt \
+                                         --output gs://YOUR_GCS_BUCKET/counts \
+                                         --runner DataflowRunner \
+                                         --project YOUR_GCP_PROJECT \
+                                         --region YOUR_GCP_REGION \
+                                         --temp_location gs://YOUR_GCS_BUCKET/tmp/
+```
+{{end}}
 
+{{if (eq .Sdk "java" "python")}}
```

Comment:
Is there a reason this is limited to Java/Python? Flink can run pipelines for any language

Reply:
Same for Spark, Samza, etc...

Reply:
Is it just because we don't have instructions for running the pipeline with Go yet?

Reply:
Yes, I created an issue to extend this information https://github.com/apache/beam/issues/25087 in scope of this item I only converted existing Introduction unit to a new template format. I

Reply:
Sounds good - thanks!